test/config_test/config_test.cc

```diff
@@ -74,11 +76,17 @@ class ConfigTest {
 };
 
 uint32_t run(const std::string& directory) {
+  // Change working directory, otherwise we won't be able to read files using relative paths.
+  char cwd[PATH_MAX];
+  RELEASE_ASSERT(::getcwd(cwd, PATH_MAX) != nullptr);
+  RELEASE_ASSERT(::chdir(directory.c_str()) == 0);
   uint32_t num_tested = 0;
-  for (const std::string& filename : TestUtility::listFiles(directory, true)) {
+  for (const std::string& filename : TestUtility::listFiles(directory, false)) {
```

Comment:
Out of curiosity, is there a reason this recursive flag was flipped to false?

Reply:
Unit test certificates are in the `certs` sub-directory, and we don't want to load configuration from them.

Reply:
See: https://github.com/PiotrSikora/envoy/commit/5e68dedcc1c743e70d1676fe2a0c1dc388d330b6#diff-4133d0a7b4e910fd0a9861deab153595

Reply:
Thanks! Missed that.